Resources/Prototypes/DeltaV/Maps/salvage_modified.yml

```diff
@@ -1,105 +1,112 @@
-﻿# Override for /Prototypes/Maps/salvage.yml
+# Override for /Prototypes/Maps/salvage.yml, see /IgnoredPrototypes/ignoredPrototypes.yml
+# No small wrecks here, only medium and large.
 
 # "Medium"-class maps - Max size square: 15x15, indicated size: 7.5
 
-- type: salvageMap
-  id: DVOutpostArm
-  mapPath: /Maps/Salvage/DeltaV/DV-outpost-arm.yml
-  size: medium
-
 - type: salvageMap
   id: DVMedium1
   mapPath: /Maps/Salvage/DeltaV/DV-medium-01.yml
-  size: medium
+  sizeString: salvage-map-wreck-size-medium
 
 - type: salvageMap
   id: DVMediumVault1
-  mapPath: /Maps/Salvage/DeltaV/DV-med-vault-01.yml
-  size: medium
+  mapPath: /Maps/Salvage/DeltaV/DV-medium-01.yml
```

Comment:
This one should still say `medium-vault` or it will duplicate the one above it

Reply:
typo ops